content/post/051-new-ipfs-release-process.md

```diff
@@ -0,0 +1,229 @@
+---
+date: 2019-07-18
+url: 2019-07-18-ipfs-release-process
+tags: IPFS, go-ipfs, release process, 
+title: Improving the IPFS Release Process
+author: Steven Allen, Alan Shaw, David Dias, Molly Mackinlay
+---
+
+go-ipfs is introducing a new release cycle and process to ensure reliable releases.
+
+## Background
+
+When reflecting on Q2, we noticed an unusual number of critical regressions
+(since fixed) introduced in the last three go-ipfs releases:
```

Comment:
I would lead with a positive - this is a good thing because...and here's the reason why we did it.

> IPFS is growing and maturing 🌱. We're seeing many more users in our network and have recognised that we need to level up our release process to deliver a more stable product, on a predictable schedule. We're doing this because in Q2 we shipped an unusual number of critical regressions 😱. Rest assured that these have since been fixed 😅. However, we don't want that to happen again, so we're putting safeguards in place to minimise the chances.
>
> Full disclosure - here's what happened:

Reply:
Updated minus the emojies (for consistency). Thanks!